sympy/combinatorics/coset_table.py

```diff
@@ -766,14 +819,80 @@ def conjugates(self, R):
             R_set.difference_update(r)
         return R_c_list
 
+    ##############################
+    #      Modified Methods      #
+    ##############################
+
+    def modified_define(self, alpha, x):
+        '''
+        Input: \alpha \epsilon \Omega, x \epsilon A*
+
+        Summary
+        =======
+        Define a function p_p from from [1..n] to A* as
+        an additional component of the modifief coset table.
+
+        See Also
+        ========
+        define
+        '''
```

Comment:
I can't see what exactly is going on from here, but Python 3.6 claims a SyntaxError due to an invalid escape sequence `\e` here. Is there a unusual character (rather than just `\newline`) after or before `'''`?

Reply:
Not able to produce that error locally. The doctests with Python3 run fine on my machine.

Reply:
@RavicharanN Is your python3 the 3.6 version though?

Reply:
Yeah, it is. As @jksuom suggested, using raw strings did the trick. :+1: